lib/task.js

```diff
@@ -120,7 +120,7 @@ Task.prototype.help = function (callback) {
   }
 
   log.info("help",
-    "\n\n" + chalk.green.bold("Usage") + ": \n\n  builder " + action + " [flags] [task]" +
+    "\n\n" + chalk.green.bold("Usage") + ": \n\n  builder " + action + " <action> <task(s)>" +
```

Comment:
Oh, `--builderrc` was the only flag listed for `builder help` when I issued that command. So leave `<flag(s)>` out of the usage then?